contrib/roles/windows/docker/tasks/requirements-16299.yaml

```diff
@@ -8,9 +8,9 @@
   register: features_installed
   with_items:
     - Containers
-    - Hyper-V # TODO
-    - RSAT-Hyper-V-Tools # TODO
-    - Hyper-V-PowerShell # TODO
+    #- Hyper-V # TODO
+    #- RSAT-Hyper-V-Tools # TODO
+    #- Hyper-V-PowerShell # TODO
 
 - name: Windows 1709 | Expect reboot_required to false
```

Comment:
Rename the filename to ``requirements.yaml`` and put ``Windows Server`` instead of ``Windows 1709`` as those requirements will be for all versions of Windows.

Reply:
Sure